spec/features/scheduling/schedule_unpublishable_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+RSpec.feature "Schedule an unpublishable edition" do
+  scenario do
+    given_there_is_an_edition_with_issues
+    when_i_go_to_schedule_the_edition
```

Comment:
Minor - I think it would help clarify this spec to separate visiting the summary page, or rename the subsequent step since we haven't actually left the page from the user's point of view ;-).